src/main/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/PodTemplateStep.java

```diff
@@ -402,7 +402,7 @@ public DescriptorImpl() {
         @SuppressWarnings("unused") // by stapler/jelly
         public ListBoxModel doFillCloudItems() {
             ListBoxModel result = new ListBoxModel();
-            result.add("-- any --", "");
+            result.add("-any-", "");
```

Comment:
FWIW I suggested
```suggestion
            result.add("—any—", "");
```
or whatever looks nice in your browser

Reply:
Ah,I didn't see that it was a different character.

Reply:
https://getitwriteonline.com/articles/en-dashes-em-dashes/ Didn't even know the _en dash_.

Reply:
And let us not forget U+2015 HORIZONTAL BAR ― which crops up a lot in Irish novels IIRC!